lib/webvalve/fake_service_wrapper.rb

```diff
@@ -1,18 +1,23 @@
 module WebValve
   class FakeServiceWrapper
     # lazily resolve the app constant to leverage rails class reloading
-    def initialize(app_class_name)
-      @app_class_name = app_class_name
+    def initialize(service_config)
+      @service_config = service_config
     end
 
     def call(env)
+      env['PATH_INFO'] = env['PATH_INFO'].gsub(/^#{path_prefix}/, '')
       app.call(env)
     end
 
     private
 
     def app
-      @app_class_name.constantize
+      @service_config.service_class_name.constantize
+    end
+
+    def path_prefix
+      @path_prefix ||= URI::parse(@service_config.service_url).path
```

Comment:
Not sure if we're parsing URLs all over the place and could consolidate onto the config. Thoughts?

Reply:
yea, it might be nice to have the config expose a `path_prefix` method instead of doing this here. it won't be used anywhere else, but it would be tidy.